### PR TITLE
Log to file during update

### PR DESCRIPTION
**Pull request by kvanhijf**

### CODE REVIEW

File: ovs/lib/albaupdate.py [hunk 1]
```diff
@@ -547,10 +554,12 @@ def _post_update_alba_plugin_alba(components):
 
         # Update ALBA nodes
         AlbaUpdateController._logger.info('Executing hook {0}'.format(inspect.currentframe().f_code.co_name))
-        for node in AlbaNodeList.get_albanodes():
-            if node.client.get_package_information():
-                AlbaUpdateController._logger.info('{0}: Restarting services'.format(node.ip))
-                node.client.restart_services()
+        alba_nodes = AlbaNodeList.get_albanodes()
+        alba_nodes.sort(key=lambda node: ExtensionsToolbox.advanced_sort(element=node.ip, separator='.'))
```
Comment (JeffreyDevloo) [inline ovs/lib/albaupdate.py]: Might give troubles with Active Drive as the generic nodes do not have an ip

File: ovs/lib/albaupdate.py [hunk 2]
```diff
@@ -547,10 +554,12 @@ def _post_update_alba_plugin_alba(components):
...
+        for alba_node in alba_nodes:
+            if alba_node.client.get_package_information():
+                AlbaUpdateController._logger.info('{0}: Restarting services'.format(alba_node.ip))
+                alba_node.client.restart_services()
```
Comment (JeffreyDevloo) [inline ovs/lib/albaupdate.py]: Calls is not available in the generic manager. So filter out the Generic nodes too

Comment (kvanhijf) [reply ovs/lib/albaupdate.py]: I just did :thinking:

Comment (JeffreyDevloo) [reply ovs/lib/albaupdate.py]: No :thinking:

**Pull request merged**